pheget/api/format.py

```diff
@@ -328,6 +329,10 @@ def query_variants(
     reader.add_filter("maf")
     reader.add_filter(lambda result: result.maf > 0.0)
 
+    if piponly:
+        reader.add_filter("pip")
+        reader.add_filter(lambda result: result.pip > 0.0)
```

Comment:
Is there any special reason why this cutoff was chosen? What is the difference between a variant with no pip data, and a variant with a pip value of 0?

In cases where code depends on hardcoded constants (like pip >0), it can be helpful to add a small comment describing how that constant was chosen.

Reply:
Because the way we currently implement appending PIP data to data points sets the default value to 0.0, this filter is just a crude way to filter the data down to only points that have non-missing PIP data - in the PIP database no points have a value of 0.

I will add a comment to note this.